community_health_file.md

```diff
@@ -0,0 +1,61 @@
+[//]: <> (github.com/codebuddies/backend/)
+## **Overview of Codebuddies**
+
+### What is CodeBuddies?
+[CodeBuddies](https://codebuddies.org/) is a remote-first community of independent code learners who enjoy sharing knowledge and helping each other learn faster via discussions and pairing. It is free and open-sourced, and supported by open source contributors and financial backers on our [Open Collective](https://opencollective.com/codebuddies).  
+
+### CodeBuddies website Version 3 (CBV3)
+
+We are currently building out a new platform (CBV3) to replace the [old version](http://github.com/codebuddies/codebuddies) which is currently shown at [codebuddies.org]((https://codebuddies.org/)).  
+
+The new version of the website will include features that will help users:
+* share and recommend resources/tutorial links
+* find open source projects to work on
+* form study groups
+* collect learning paths and share them
+* pair programming -- find 1-hour pair programming partners for any topics in a much more easier way
+
+## **Role of Backend**
+The backend of CBV3 is an app which is composed of Django REST Framework that provides API endpoints which the frontend app (React) consumes.
```

Comment:
Can you tweak this to be:

The backend is a Django app that provides API endpoints (built using Django REST Framework) that the frontend (a React app) consumes.

Reply:
just a quick question/clarification on some technical terms 
I've always thought Django is a framework that coders can write apps within the framework. Do the definitions of app and framework are similar in this case? 
I'm not too familiar with applications and frameworks let me know thank you :)

Reply:
Good question! Django by itself is a framework, you're right! We're using a package called Django REST Framework to make our API. I called it a "Django app" in the sense that it is a web app, just as any sort of website is an app to me. I wasn't thinking of the technical definition of `app` inside Django per se. :) 

Maybe to be clearer:

The backend is a Django application that provides API endpoints (built using [Django REST Framework](https://github.com/encode/django-rest-framework)) that the frontend (a React app) consumes.